src/hw_isolation_event/hw_status_manager.cpp

```diff
@@ -23,8 +32,14 @@ namespace fs = std::filesystem;
 constexpr auto HW_STATUS_EVENTS_PATH =
     HW_ISOLATION_OBJPATH "/events/hw_isolation_status";
 
-Manager::Manager(sdbusplus::bus::bus& bus) : _bus(bus), _lastEventId(0)
-{}
+Manager::Manager(sdbusplus::bus::bus& bus,
+                 record::Manager& hwIsolationRecordMgr) :
+    _bus(bus),
+    _lastEventId(0), _isolatableHWs(bus),
+    _hwIsolationRecordMgr(hwIsolationRecordMgr)
+{
+    _requiredHwsPdbgClass = {"dimm", "fc"};
```

Comment:
this can as part of initializer list or else we have two operations 1) constructor for _requiredHwsPdbgClass and 2) operator= for {"dimm", "fc"};

Reply:
I will agree with you on the costly operation.

I thought in the future we might add more units so I thoughts to keep it inside the constructor itself but, I will move it to as part of the initializer list and later I will revisit with some better approaches while adding more units.